doc/runtime-information.rst

```diff
@@ -101,6 +101,36 @@ symbolic link::
 
 
 
+LD_LIBRARY_PATH / LIBPATH considerations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This environment variable is used to discover libraries, it is the library
+search path - on Linux and *BSD `LD_LIBRARY_PATH` is used, on AIX it is
+`LIBPATH`.
+
+PyInstaller saves the original value to `*_ORIG`, then modifies the search
+path so that the bundled libraries are found first by the bundled code.
+
+But if your code executes a system program, you often do not want that this
```

Comment:
Many thanks for this doc update :-)

I would put this paragraph at the start of the section, so those who don't have this use case would not need to read further.